CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest
+- Changed default resolution of Camera2VideoSourceImpl to 960*540
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Removed deprecated code for 0.12.0
+- Replaced `VisionView` rendering with common rendering engine for all platforms
+- Replaced `VisionArView` rendering with common rendering engine for all platforms
+- Integrated LaneDetection mode for VisionView
```

Comment:
BTW has iOS implemented the same? @kiryldz @chezzdev

Reply:
No, since there's no VRE integrated, we don't have this mode.